SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverRPC.cpp

```diff
@@ -0,0 +1,505 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialNetDriverRPC.h"
+#include "EngineClasses/SpatialNetBitReader.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "Interop/Connection/SpatialTraceEventBuilder.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Utils/RPCRingBuffer.h"
+#include "Utils/RepLayoutUtils.h"
+
+#include "Interop/RPCs/RPCQueues.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialNetDriverRPC);
+
+using namespace SpatialGDK;
```

Comment:
We don't seem to use this using?

Reply:
We are probably using it in the methods bodies, for convenience.
Signature are copy/pasted from the header, hence why they repeat the namespace, and I do not think that is an issue?